web/signature_manager.css

```
@@ -0,0 +1,598 @@
+            stroke-miterlimit: 10;
+
+            &:hover {
+              cursor: var(--draw-cursor);
+            }
+          }
+
+          #thickness {
+            position: absolute;
+            width: 100%;
+            inset-block-end: 0;
+            display: grid;
+            align-items: center;
+            justify-content: center;
+            pointer-events: none;
+
+            > span {
+              color: var(--signature-draw-placeholder-color);
+            }
+
+            > div {
+              width: auto;
+              height: auto;
+              display: flex;
+              align-items: center;
+              justify-content: center;
+              gap: 8px;
+              padding: 6px 8px;
+              margin: 0;
+              background-color: var(--thickness-bg);
+              border-radius: 4px 4px 0 0;
+              pointer-events: auto;
+
+              > label {
+                color: var(--thickness-label-color);
+              }
+
+              > input {
+                width: 100px;
+                height: 14px;
+                background-color: transparent;
+
+                /*#if !MOZCENTRAL*/
+                &::-webkit-slider-runnable-track,
+                /*#endif*/
+                &::-moz-range-track,
+                &::-moz-range-progress {
+                  background-color: var(--thickness-slider-color);
+                }
+
+                /*#if !MOZCENTRAL*/
+                &::-webkit-slider-thumb,
+                /*#endif*/
+                &::-moz-range-thumb {
+                  background-color: var(--thickness-bg);
+                }
+
+                border-radius: 4.5px;
+                border: 0;
+                color: var(--signature-color);
+              }
+            }
+          }
+        }
+
+        &#addSignatureImageContainer {
+          display: none;
+
+          > svg {
+            stroke: none;
+            stroke-width: 0;
+            fill: var(--primary-color);
+            fill-opacity: 1;
+          }
+
+          #addSignatureImagePlaceholder {
+            position: absolute;
+            top: 0;
+            left: 0;
+            width: 100%;
+            height: 100%;
+            background-color: transparent;
+            display: flex;
+            flex-direction: column;
+            align-items: center;
+            justify-content: center;
+
+            a {
+              text-decoration: underline;
+              cursor: pointer;
+            }
+          }
+
+          #addSignatureFilePicker {
+            visibility: hidden;
+            position: relative;
+            width: 0;
+            height: 0;
+          }
+        }
+      }
+
+      &[data-selected="type"] > #addSignatureTypeContainer {
+        display: block;
+      }
+
+      &[data-selected="draw"] > #addSignatureDrawContainer {
+        display: block;
+      }
+
+      &[data-selected="image"] > #addSignatureImageContainer {
+        display: block;
+      }
```

Could these be combined, or does that not work?
```suggestion
      &[data-selected="type"] > #addSignatureTypeContainer,
      &[data-selected="draw"] > #addSignatureDrawContainer,
      &[data-selected="image"] > #addSignatureImageContainer {
        display: block;
      }
```